server/settings/tools.py

```diff
@@ -448,6 +448,17 @@ class GlobalToolsModel(BaseSettingsModel):
                 "task_types": [],
                 "tasks": [],
                 "template": "SK_{folder[name]}{variant}"
+            },
+            {
+                "product_types": [
+                    "hda"
+                ],
+                "hosts": [
+                    "houdini"
+                ],
+                "task_types": [],
+                "tasks": [],
+                "template": "{folder[name]}_{variant}"
```

Comment:
Wouldn't this technically still make it conflict on folder name? What did this profile solve again? Sorry, I seem to be suffering from short-term memory loss this week.

Reply:
It doesn't solve anything it's just a regular request. 
I used folder name instead of folder label following onderj's screenshot https://github.com/ynput/ayon-core/pull/456#discussion_r1638156081

> ![image](https://github.com/ynput/ayon-core/assets/33513211/105927ae-18b0-46e0-b550-59176469ad3e)

Reply:
Why is it requested? What does the request solve? Seems like an odd default setting to me if it's not there to avoid a certain issue but is a specific studio request? :)

Reply:
> tbh, supporting the folder name was not particularly requested, however, it's odd that the folder name exists in many templates but not HDA.

It works in the publish filename templates by default, just not the product name itself which by default does not get this data indeed. If it's truly not needed for the HDA logic to work I'd remove it from this PR.

If there's a 'generic' need for `{folder[name]}` to be accessible for product name profiles by default then likely we're better off solving this differently so that it works across the board instead of fixing a particular HDA only case?

Reply:
I don't know about the `generic` need. maybe it's a question for @antirotor or @iLLiCiTiT 
---
> If it's truly not needed for the HDA logic to work I'd remove it from this PR.

We still need a default template profile for HDAs. maybe let's stick with the default `{product[type]}{variant}`
and we can wait till some complains about using `{asset}` or `{folder[name]}`

---
Also, I've just discovered that `{folder[name]}` key is being replaced with `{asset}` therefore the creator prints an error message. 
```
    raise TemplateFillError(
ayon_core.pipeline.create.product_name.TemplateFillError: Value for 'asset' key is missing in template '{asset}{variant}'. Available values are {'variant': 'Main', 'family': 'hda', 'task': {'name': 'cfx', 'type': 'CFX'}, 'product': {'type': 'hda'}, 'folder': {'label': 'Moco', 'name': 'moco'}}
```
![image](https://github.com/ynput/ayon-core/assets/20871534/afb3f667-588e-4aad-93a8-d98bb6c0a8ed)

Reply:
> We still need a default template profile for HDAs

Could you explain why we need it? Why wouldn't the normal default publish template work for HDAs? Again, what does the custom template solve?

Reply:
> Could you explain why we need it? Why wouldn't the normal default publish template work for HDAs? Again, what does the custom template solve?

From a chat with @antirotor, 
```
We will probably need to create new product name template for those HDA so they will have more meaningful names in this context.
```

Reply:
Thanks. I still feel like I'm missing the crucial bit of the puzzle. Is it the case that the filename defines the HDA node name or type name or whatever and that's why we care so much about the product naming specifically for the HDA compared to other products?

I think I'm lacking the info on why there's this need to differ for published HDAs.

Reply:
Personally, I think it's about the HDA name itself. e.g. in this tab menu, I'd prefer to prefix my HDA names with folder name to make it easy for me to identify them and I don't like the default `hda` prefix. 
![image](https://github.com/ynput/ayon-core/assets/20871534/a913f665-9244-48db-87e7-63180dbd1023)

But, It's better to leave it question for @antirotor.

Reply:
Mentioning @moonyuet for reference.